File: Game/server/Game/Item/Horse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Game.Item
{
    public class Horse : Item
    {
        private BHorseExtra extra;

        public Horse(ContainerWithOne container, BHorseExtra extra) : base(container)
        {
            this.extra = extra;
        }

        public int Speed => extra.Speed;

        public override void Use()
        {
            
        }
    }
}

File: ZezeJava/ZezexJava/server/src/Game/Item/Horse.java
package Game.Item;

public class Horse extends Item {
	private final BHorseExtra extra;

	public Horse(Game.Equip.BItem bItem, BHorseExtra extra) {
		super(bItem);
		this.extra = extra;
	}

	public final int getSpeed() {
		return extra.getSpeed();
	}

	@Override
	public boolean Use() {
		return false;
	}
}
